src/main/java/training/Elbow/ElbowConstants.java

```diff
@@ -0,0 +1,25 @@
+package training.Elbow;
+
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class ElbowConstants {
+    //* These constants are temp and shall be replaced with calibrated constants that aren't 0
+    protected static final String ELBOW_LOG_PATH = "";
+    protected static final Rotation2d DEFAULT_POSITION_ELBOW = new Rotation2d(1, 1);
```

Comment:
in the other positions you didnt write _ELBOW. and try to group the related ones together!